controllers/users.js

```diff
@@ -28,9 +27,9 @@ router.post('/', function(req, res) {
 // Update a user
```

Comment:
There is currently zero checking of the fields they are sending up to us. Zero sanitizing in general